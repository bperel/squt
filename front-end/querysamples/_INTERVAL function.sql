SELECT INTERVAL(1,2,3,4) AS my_interval