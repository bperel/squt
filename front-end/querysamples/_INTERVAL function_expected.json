{
	"Functions" : {
		"my_interval" : {
			"to" : "OUTPUT",
			"name" : "interval",
			"alias" : "my_interval",
			"Constants" : {
				"1" : "1",
				"2" : "2",
				"3" : "3",
				"4" : "4"
			}
		}
	}
}
